lib/galaxy/model/__init__.py

```diff
@@ -78,6 +78,11 @@
 AUTO_PROPAGATED_TAGS = ["name", "group"]
 
 
+class RepresentById(object):
+    def __repr__(self):
+        return '<galaxy.model.%s(%s)>' % (self.__class__.__name__, self.id)
```

Comment:
This will fire a DB query after a flush for every log statement that includes a model object. Consider using cached_id in this file instead (https://github.com/galaxyproject/galaxy/pull/6524/files#diff-5805bf58afdc31ee4a9c4e788301b11dR163).